contracts/interfaces/IEnum.sol

```diff
@@ -3,9 +3,9 @@ pragma solidity >=0.7.0 <0.9.0;
 
 /**
  * @title Enum - Collection of enums used in Safe contracts.
- * @author Richard Meissner - @rmeissner
+ * @author @safe-global/safe-protocol
  */
-abstract contract Enum {
+interface Enum {
```

Comment:
Nit: this isn't an interface - personally, I would make it a `library`.

Reply:
Done 👍🏾